internal/resources/fabric/route_filter/resource_test.go

```diff
@@ -0,0 +1,95 @@
+package route_filter_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/equinix/equinix-sdk-go/services/fabricv4"
```

Comment:
Not sure why it didn't show up as a comment, but the linter is failing because the imports in this file don't match what's expected by `goimports`